download-dependencies.sh

```diff
@@ -2,7 +2,9 @@
 
 echo "Creating directory"
 mkdir -p ./dependencies && \
-cd ./dependencies && \
+cd ./dependencies
+# Removing prior dependencies
+rm -rf * 
```

Comment:
We don't explicitly [set -e](https://stackoverflow.com/a/9952249) for this script, which means without `&&` chaining there could potentially be circumstances where `cd` fails and `rm -rf *` removes the contents of some other directory: I'd rather we do something safer?